optaweb-tsp-planner/src/main/java/org/optaweb/tsp/optawebtspplanner/TspMapController.java

```diff
@@ -58,7 +58,7 @@ public void create(Place place) {
     public void demo() {
 //        repository.findAll().forEach(planner::removePlace);
 //        repository.deleteAll();
```

Comment:
I know it's not a part of this PR, but if these lines of code are not used, let's erase them

Reply:
+1 I'll push it directly to master.